authors/apps/authentication/views.py

```diff
@@ -1,23 +1,29 @@
+import os
+from datetime import datetime, timedelta
+
+import jwt
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode, urlsafe_base64_decode
+from jwt import ExpiredSignatureError
 from rest_framework import status
-from rest_framework.generics import (RetrieveUpdateAPIView, CreateAPIView,
-                                     ListAPIView)
+from rest_framework.generics import (
+    RetrieveUpdateAPIView, CreateAPIView, UpdateAPIView, ListAPIView)
```

Comment:
The closing brace should be in a new line as done in the import statement spanning over lines 23 - 26.